src/standalone/Soil/boundary_conditions.jl

```diff
@@ -576,27 +577,40 @@ to simulate surface and subsurface runoff and this is accounted
 for when setting boundary conditions. The default is to have no runoff
 accounted for.
 
+Finally, because this same boundary condition type is used for the soil
+in integrated land surface models, we also provide a tuple
+indicating the prognostic land components present, as these affect how the boundary
+conditions are computed. The default is
+a tuple containing only (:soil,), indicating a standalone soil run.
```

Comment:
Can you add more examples of values? It it a tuple of Symbols? Which symbols are allowed?

Reply:
I noted here that it is a tuple of symbols, and referred to the documentation page I am adding.